src/routes/(app)/checkout/+page.server.ts

```diff
@@ -473,8 +474,12 @@ export const actions = {
 						userAlias: locals.user?.alias,
 						npub: locals.npub,
 						email: locals.email,
-						userHasPosOptions: locals.user?.hasPosOptions
-					},
+						secondaryEmails: locals.sso
+							?.map((sso) => sso.email)
+							.filter((email) => email !== undefined),
+						userHasPosOptions: locals.user?.hasPosOptions,
+						ssoIds: locals.sso?.map((sso) => sso.id)
```

Comment:
I wouldn't add ssoIds. I think it's not useful, since it's an internal identifier to some SSO. If the SSO is later removed (or worse, replaced by a different one) the id will be useless.

If you have a good idea why we should keep it, we could keep it (perhaps namespaced by the sso).

Reply:
> If the SSO is later removed (or worse, replaced by a different one) the id will be useless.

Yes removing a SSO will remove access for all users that used it exclusively to log in, it should not be done lightly.

Fixed the first commit to just use `userIdentifer`, and good point about the provider, added a second commit to prefix SSO ids with the provider. (there's a temporarily potential edge case if the sso ids contain `:` but wholly harmless IMO)